server/utils/lostBedUtils.ts

```diff
@@ -0,0 +1,18 @@
+import type { TemporaryAccommodationLostBed } from '@approved-premises/api'
+
+export const allStatuses: Array<{ name: string; id: TemporaryAccommodationLostBed['status']; tagClass: string }> = [
+  {
+    name: 'Active',
+    id: 'active',
+    tagClass: 'govuk-tag--green',
+  },
+  {
+    name: 'Cancelled',
+    id: 'cancelled',
+    tagClass: 'govuk-tag--red',
+  },
+]
+
+export const lostBedStatusHtml = (status: string, colour: string): string => {
```

Comment:
Heads up that I've renamed similar functions to `statusTag`, to match CAS1 naming of equivalent functions we've pulled over